extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItem.java

```diff
@@ -125,7 +126,8 @@ private static String getMimeType(GoogleMediaItem mediaItem) {
   // Guesses the mimetype from the filename, or returns null on failure.
   private static String guessMimeTypeFromFilename(String filename) {
     try {
-      return Files.probeContentType(new File(filename).toPath());
+      Tika tika = new Tika();
```

Comment:
is creating this expensive? should we make it a static final public variable instead?

Reply:
Do we also want/need to change the default type thats returned if both this and mediaItem.getMimeType() are empty?

Reply:
Made it public static final! And I think the default types are fine, but if there are more global mime-types we can use for defaults I'm happy updating it

Reply:
LGTM -- I just wasn't sure if there was a better default to use or if leaving empty was a better solution so that it can be handled on the importer side but thats a conversation we can have later as well